CONTRIBUTING.md

```diff
@@ -9,7 +9,10 @@ To start the local site use:
 To deploy updated versions:
 `npm run deploy`
 
+# Using Cloudflare Workers
 
+1. Run `npx wrangler dev src/index.js` in your terminal to start a development server
```

Comment:
The script on line 14 has an outdated path. It should be either: `npx wrangler dev ./worker/index.js` or you can replace the `dev` script with `npm run auth:worker`.

Reply:
Or you could make it more generic: `npx wrangler dev <filepath>` in case we decide to add more `workers` later.